transformer/dockerfilegenerator/windows/webappdockerfilegenerator.go

```diff
@@ -34,19 +34,43 @@ import (
 )
 
 const (
-	buildStageC           = "dotnetwebbuildstage"
-	defaultBuildOutputDir = "bin"
+	buildStageC             = "dotnetwebbuildstage"
+	defaultBuildOutputDir   = "bin"
+	defaultRunStageImageTag = "4.8"
 )
 
+var (
+	// imageTagToSupportedVersions is a mapping from image tag to dot net framework versions that image supports.
+	// Version compatibility table taken from https://hub.docker.com/_/microsoft-dotnet-framework-aspnet
+	imageTagToSupportedVersions = map[string][]string{
```

Comment:
Can we read this from a config file similar to how we do in java

Reply:
The advantage it provides is that it makes the mapping configurable without the need to change code.

Reply:
done